client/store/modules/comments/actions.js

```diff
@@ -1,14 +1,21 @@
 import generateActions from '../../helpers/actions';
+import map from 'lodash/map';
+import pickBy from 'lodash/pickBy';
 import SSEClient from '../../../SSEClient';
 
 const { api, get, save, setEndpoint, update } = generateActions();
 let SSE_CLIENT;
 
+const filterExistingComment = (existingComments, fetchedComments) => {
+  const existingId = map(existingComments, 'id');
+  return pickBy(fetchedComments, it => !existingId.includes(it.id));
+};
```

Comment:
I suggest doing this with `lodash/find` instead of `lodash/map`. 

```js
const filterExistingComment = (existingComments, fetchedComments) => {
  return pickBy(fetchedComments, it => !find(existingComments, { id: it.id }));
};
```